src/curated-corpus/pages/SchedulePage/SchedulePage.tsx

```diff
@@ -27,17 +28,28 @@ import {
 } from '../../../_shared/hooks';
 import { useStyles } from './SchedulePage.styles';
 import { DropdownOption } from '../../helpers/definitions';
+import { DatePicker } from '@material-ui/pickers';
+import { getLocalDateTimeForGuid } from '../../helpers/helperFunctions';
 
 export const SchedulePage: React.FC = (): JSX.Element => {
-  const { showNotification } = useNotifications();
   const classes = useStyles();
 
-  // set up initial start/end dates for the query
-  const [startDate, setStartDate] = useState<DateTime>(DateTime.local());
-  const [endDate, setEndDate] = useState<DateTime>(
+  /**
+   * ##########
+   * ########## State variables start here
+   * ##########
+   */
```

Comment:
I've added these comments and moved around / grouped similar pieces of logic so that it's a bit easier to read. Let me know your thoughts if this is unnecessary?

Reply:
I think it's great! 👍

Reply:
Still think it's great, however there may be some rather painful times ahead with merging the recent changes that took the word `Curated` out of most queries and mutations 😵‍💫